src/js/dataMapping/search/tableSearchFields.js

```diff
@@ -1,13 +1,11 @@
 const tableSearchFields = {
     columnWidths: {
-        award_id: 160,
-        recipient_name: 310,
-        period_of_performance_start_date: 150,
-        period_of_performance_current_end_date: 150,
-        total_obligation: 220,
-        type: 230,
-        awarding_agency_name: 280,
-        awarding_subtier_name: 280,
+        award_id: 120,
```

Comment:
Please restore the wider column widths. These are necessary to prevent the header cells from wrapping on fallback fonts.